tests/backup/backup_restore_basic_test.go

```diff
@@ -3548,3 +3553,172 @@ var _ = Describe("{BackupNamespaceInNfsRestoredFromS3}", func() {
+					secondBackupLocationUID = uuid.New()
+					err = CreateNFSBackupLocation(secondBackupLocationName, secondBackupLocationUID, orgID, "", getGlobalBucketName(provider), true)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Creating NFS backup location [%s]", secondBackupLocationName))
+					backupLocationMap[secondBackupLocationUID] = secondBackupLocationName
+				} else {
+					// Creating cloud cred again because in case of NFS as provider, cloud cred will not be created above
+					err = CreateCloudCredential("aws", s3CloudCredName, s3CloudCredUID, orgID, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential:[%s] for org [%s] for s3 backup location", s3CloudCredName, orgID))
+					secondBackupLocationName = fmt.Sprintf("%s-%s-%v", "s3", getGlobalBucketName(provider), RandomString(5))
+					secondBackupLocationUID = uuid.New()
+					err = CreateS3BackupLocation(secondBackupLocationName, secondBackupLocationUID, s3CloudCredName, s3CloudCredUID, getGlobalBucketName(provider), orgID, "")
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of S3 backup location [%s]", secondBackupLocationName))
+					backupLocationMap[secondBackupLocationUID] = secondBackupLocationName
+				}
+			}
+		})
+
+		Step("Registering application clusters for backup", func() {
+			log.InfoD("Registering application clusters for backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = CreateApplicationClusters(orgID, "", "", ctx)
+			dash.VerifyFatal(err, nil, "Creating source and destination cluster")
+			log.InfoD("Verifying cluster status for both source and destination clusters")
+			clusterStatus, err := Inst().Backup.GetClusterStatus(orgID, SourceClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", SourceClusterName))
+			srcClusterUid, err = Inst().Backup.GetClusterUID(ctx, orgID, SourceClusterName)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching [%s] cluster uid", SourceClusterName))
+			clusterStatus, err = Inst().Backup.GetClusterStatus(orgID, destinationClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", destinationClusterName))
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", destinationClusterName))
+		})
+
+		Step("Create schedule policy", func() {
+			log.InfoD("Creating schedule policy")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			schedulePolicyName = fmt.Sprintf("%s-%v", "periodic-schedule-policy", RandomString(5))
+			schedulePolicyInfo = Inst().Backup.CreateIntervalSchedulePolicy(5, schedulePolicyInterval, 5)
```

Comment:
can make use of CreateBackupScheduleIntervalPolicy here

Reply:
done